crates/orderbook/src/main.rs

```diff
@@ -509,6 +511,12 @@ async fn main() {
                         }),
                         &estimator.name(),
                     )),
+                    PriceEstimatorType::OneInch => Box::new(instrumented(
```

Comment:
```suggestion
                    PriceEstimatorType::OneInch => Box::new(instrumented_and_cached(
```

Yesterday I introduced this cache for the other external price estimators. It probably makes sense for 1Inch too.